compiler/packages/babel-plugin-react-compiler/src/HIR/Environment.ts

```diff
@@ -794,6 +797,21 @@ export class Environment {
               binding.imported,
             );
             if (importedType != null) {
+              /*
```

Comment:
Maybe factor this out into a helper?

Reply:
Hmm it's kind of tedious to get customize the error message if we extract to a helper, i'm gonna leave it